backend/vault.go

```diff
@@ -33,7 +35,8 @@ func (c *client) vaultLogin() error {
 		"role_id":   c.roleID,
 		"secret_id": c.secretID,
 	}
-	resp, err := c.logical.Write("auth/approle/login", appRole)
+	approleLoginPath := fmt.Sprintf("auth/%s/login", c.approlePath)
+	resp, err := c.logical.Write(approleLoginPath, appRole)
```

Comment:
I think we can squash these 2 lines in just 1, wdyt?

Reply:
> I think we can squash these 2 lines in just 1, wdyt?

ok 👍